colossalai/inference/inference_state.py

```diff
@@ -0,0 +1,51 @@
+# might want to consider combine with InferenceConfig in colossalai/ppinference/inference_config.py later
+from dataclasses import dataclass
+from typing import Any
+
+import torch
+
+from colossalai.inference.kvcache_manager import MemoryManager
+
+
+@dataclass
```

Comment:
why not use infer_info :)

Reply:
To refactor the class and add other attributes we would like to use as well as some utility functions